code-env/python/spec/requirements.txt

```diff
@@ -1,8 +1,11 @@
-pdf2image==1.14.0
+pypdfium2==4.17.0
```

Comment:
I'm using the latest version of pypdfium2 release just this morning because the previous version (4.16.0) was broken on python 3.6 on centos (here is a screenshot of the weird error on dku34):

<img width="1271" alt="Screenshot 2023-06-27 at 11 31 28" src="https://github.com/dataiku/dss-plugin-tesseract-ocr/assets/60091564/9bb3d93d-ea70-4ba6-9cb3-3d7821703e9f">